dashboards/k8s-storage-resources-glusterfs-pv.libsonnet

```diff
@@ -0,0 +1,291 @@
+local g = import 'grafonnet/grafana.libsonnet';
+
+{
+  grafanaDashboards+:: {
+    'k8s-storage-resources-glusterfs-pv.json':
+      local tableStyles = {
+        fop: {
+          alias: 'File Operations',
+        },
+      };
+      local statPanelArgs = {
+        height: '75px',
+        span: 3,
+      };
+      local brickStatPanelArgs = {
+        height: '75px',
+        span: 4,
+      };
+      local graphPanelArgs = {
+        span: 6,
+      };
+
+      // Following panels to be added in Volume Summary after IOPS panel (One metrics are availabe)
+      // .addPanel(
+      //     g.panel('Rebalance-Files') +
+      //     statPanelArgs +
+      //     g.statPanel('')
+      //   )
+      //   .addPanel(
+      //     g.panel('Rebalance-Size') +
+      //     statPanelArgs +
+      //     g.statPanel('')
+      //   )
+      //   .addPanel(
+      //     g.panel('Rebalance-Skipped') +
+      //     statPanelArgs +
+      //     g.statPanel('')
+      //   )
+      //   .addPanel(
+      //     g.panel('Rebalance-Failed') +
+      //     statPanelArgs +
+      //     g.statPanel('')
+      //   )
+      // The below variables needs to chanaged when cluster_id is introduced in the metrics
+      // local queryLabels = "{job='$OCS_Cluster', volume='$Volume'}";
+      // local fopQueryLabel = "{job='$OCS_Cluster', volume='$Volume', fop!=''}";
+      // Below variables are defined considering that there is only one gluster cluster.
+      local queryLabels = "{volume='$Volume'}";
+      local fopQueryLabel = "{volume='$Volume', fop!=''}";
+      local fopLockQueryLabel = "{volume='$Volume',fop='LOCK_OPS'}";
+      local fopInodeQueryLabel = "{volume='$Volume',fop='INODE_OPS'}";
+      local fopEntryQueryLabel = "{volume='$Volume',fop='ENTRY_OPS'}";
+      local fopReadWriteQueryLabel = "{volume='$Volume',fop='READ_WRITE_OPS'}";
+
+      // g.dashboard(
+      //   'K8s / Storage Resources / Persistent Volume / glusterfs',
+      //   uid=($._config.grafanaDashboardIDs['k8s-storage-resources-glusterfs-pv.json']),
+      // ).addTemplate('OCS_Cluster', 'gluster_volume_up', 'job')
+      // .addTemplate('Volume', 'gluster_volume_up{job = "$OCS_Cluster"}', 'volume')
+      g.dashboard(
+        'K8s / Storage Resources / Persistent Volume / glusterfs',
+        uid=($._config.grafanaDashboardIDs['k8s-storage-resources-glusterfs-pv.json']),
+      )
+      .addTemplate('Volume', 'gluster_volume_up', 'volume')
+      .addRow(
+        (g.row('Volume Summary') +
+         {
+           showTitle: false,
+           panels: self._panels,
+         })
+        .addPanel(
+          g.panel('Status') +
+          statPanelArgs +
+          g.statPanel('gluster_volume_up{volume = "$Volume"}',) +
```

Comment:
why `,` in the last of the line? is it required?

Reply:
It is not required but no harm as well. Will remove it.